draft-ietf-rats-corim.md

```diff
@@ -654,7 +706,7 @@ The following describes each member of the `triples-map`:
 
 ##### Environment
 
-An `environment-map` may be used to represent a whole attester, an attesting
+An `environment-map` may be used to represent a whole Attester, an attesting
```

Comment:
```suggestion
An `environment-map` may be used to represent a whole Attester, an Attesting
```

Reply:
@nedmsmith This section was not modified but as this comment is a logical comment, I have changed it in the latest version!